tests/buffer/global_buffer_set_final_data.cpp

```diff
@@ -9,38 +9,35 @@
 using namespace cl::sycl;
 
 
-constexpr size_t N = 16;
 
-buffer<int> b { N };
 
-int test_main(int argc, char *argv[]) {
+int test_main(int /*argc*/, char** /**argv[]*/) {
+  constexpr size_t N = 16;
+  
+  buffer<int> buf{ N };
```

Comment:
Apologies, it was my understanding "global" refers to the global memory namespace the buffer resides in. It does make more sense this way however. :)

I will rename the variable to something more unique, because somewhere inside the headers, there is a local variable named `buf` and that's what triggers the warning. Most likely I will rename the header version to something more unique and leave nice names like `b` and `buf` for user code.